oci/spec_opts.go

```diff
@@ -1653,6 +1653,10 @@ func WithCDIDevices(devices ...string) SpecOpts {
 			return fmt.Errorf("CDI device injection failed: %w", err)
 		}
 
+		if _, err := registry.UpdateMounts(s, mounts...); err != nil {
```

Comment:
As I called out in https://github.com/cncf-tags/container-device-interface/pull/169 this function is incorrectly named since from a client perspective, we're still injecting devices here -- ignoring errors if these are not valid device names.

Maybe a follow-up question: at which point do we expect these Devices to exist, and when is a failure to resolve them an error?

Reply:
What I mean is that calling these *Mounts* from a CDI perspective is incorrect. The top-level entity that CDI understands is *Devices*. A CDI Device (note the capital D) is a collection of low-level entities (device nodes, mounts, environment variables, and container lifecycle hooks) that are required to provide access to the named resource.

The fact that we are using the name Mounts here indicates a mismatch across this API.

Reply:
Ok, The interface `UpdateMounts` is removed.